index.html

```diff
@@ -1,7 +1,6 @@
 ---
 title: Meshery | The Service Mesh Management Plane
-description: Quickly compare and learn service meshes like Istio, Linkerd, Envoy, Octarine, App Mesh, Consul, Kuma, Maesh, Tanzu.
-Lifecycle, performance and configuration management across different service meshes. Patterns and best practices of service mesh operation.
+description: Quickly compare and learn service meshes like Istio, Linkerd, Envoy, Octarine, App Mesh, Consul, Kuma, Maesh, Tanzu. Lifecycle, performance and configuration management across different service meshes. Patterns and best practices of service mesh operation.
```

Comment:
What do you think about adding “osm” to those listed in the description?

Reply:
Great suggestion, @PluckyPrecious. Yes, totally.

Reply:
Added OSM to the page description.